client/src/components/add-username-modal/add-username-modal.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../scss/variables';
+@import '../../scss/colors';
+@import '../../scss/mixins';
+@import '../../scss/atoms/button';
+
+@import '../comment/comment.scss';
+
+.username-modal {
+  @include z-level(3);
+  display: flex;
+  flex-flow: column;
+  justify-content: space-around;
+  position: fixed;
+  right: $baseline;
+  bottom: $baseline;
+  width: 35vw;
+  height: 15vh;
+  border: solid $palette-light 1px;
+  border-radius: $border-radius-default;
+  background-color: $palette-lighter;
+  cursor: default;
+  &:focus {
+    outline: none;
+    box-shadow: 0 2px 10px $palette-shadow-default;
+  }
+}
+
+.add-button {
```

Comment:
cahnge to input[type="submit"], could it be extended from some place => even DRY:er?

Reply:
fixable later